### DISCUSSION TITLE
Export screen as graphic (png)

### DISCUSSION THREAD

**Discussion by mexp2** [sbrin/lopaka#174] [Ideas]
It would be great to be able to export the designed screens as graphics, e.g. png. Ideally with a scaling factor (100% 150% ....)

**Comment by sbrin** [comment]
Thanks, that's quite easy to implement. Just out of curiosity: could you describe the use case for such scenario?
Why you need that PNG? Why you need it scaled?

**Reply by mexp2** [reply]
I would like to use the screens for mockups in paper form and use them in documentation.
The scaling would be very helpful because if I have a screen with 128x64 pixels, that would be a pretty small graphic.
The tool itself is more than helpful if you just want to try out different variants without going straight to the real embedded system
